submissions/chabuChabu/database/trips.sql

```diff
@@ -0,0 +1,37 @@
+/*The following query is used to create a table for storing passanger details*/
+use trips;
+create table passangers(
+id int not null auto_increment,
+fullname varchar(100) not null,
+sex varchar(10) not null,
+age int(11),
```

Comment:
use `varchar` instead

Reply:
@adaugochi which line are you referring too?

Reply:
Line 7 @ChabuChabu

Reply:
Line 7